src/pl/plr.spec

```diff
@@ -3,10 +3,14 @@ Name:           plr-hawq
 Version:        08.03.00.14
 Release:		0
 Prefix:         /usr/local
-License:		GPL
+License:		Apache
+BuildRequires:  R-devel >= 3.1.0
```

Comment:
How about removing the version requirement, if it works fine with R on current rhel 6.x and rhel7?